dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -26,6 +26,8 @@ OpenFileFailed              =             failed to open file %1%
 NoJobDefined                =             no job found in the .jobs file
 IncorrectDelay              =             inconsistent delay %1% at time %2% (max delay is %3%)
 IterationStepAndTimeStepBothDefined =     iteration step and time step can't be defined at the same time
+NoTimelineDefined           =             timeline is not defined
```

Comment:
This is not an error case, the message should simply not be dumped

Reply:
What error should be thrown here instead ? : 

```
if (timeline != nullptr) {
    constexpr size_t failingCriteriaNumber = 5;
    const size_t maxFailingCritArraySize = std::min(failingCriteriaNumber, failingCriteriaVec.size());
    for (size_t timelineIndex = 0; timelineIndex < maxFailingCritArraySize; ++timelineIndex) {
      failingCriteriaVec[timelineIndex]->printOneFailingCriteriaIntoTimeline(timeline, currentTime);
    }
  } else {
    throw DYNError(Error::SIMULATION, NoTimelineDefined);
  }
```